systemd/openqa-webui.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=The openQA web UI
 Wants=openqa-setup-db.service openqa-livehandler.service openqa-websockets.service openqa-gru.service openqa-enqueue-asset-cleanup.timer openqa-enqueue-result-cleanup.timer openqa-enqueue-bug-cleanup.timer openqa-minion-restart.path
-After=postgresql.service openqa-setup-db.service openqa-scheduler.service nss-lookup.target remote-fs.target
+After=postgresql.service openqa-setup-db.service openqa-scheduler.service nss-lookup.target remote-fs.target apache2.service nginx.service caddy.service
```

Comment:
Wouldn't this allow a short period on startup where a webserver is running but cannot serve requests because the backend is still starting up?

Reply:
Given startup seems to be *much* quicker than shutdown, I think it is an acceptable trade-off. But it would still be nice to have both I guess?